script.js

```diff
@@ -1,117 +1,7 @@
-const rootElem = document.getElementById("root");
-const searchDiv = document.createElement("div");
-const episodeDiv = document.createElement("div");
-
-searchDiv.className = "searchDiv row";
-episodeDiv.className = "episodeDiv row";
-
-const selectEpisode = document.createElement("select");
-const selectShow = document.createElement("select");
-const searchText = document.createElement("input");
-const searchState = document.createElement("label");
-
-selectShow.innerHTML = `<option>All shows</option>`;
-selectEpisode.innerHTML = `<option>All Episodes</option>`;
-searchText.placeholder = "search...";
-
-searchText.type = "text";
-
-selectShow.className = "select xl-col-2 lg-col-2 md-col-3 sm-col-4 col-10";
-selectEpisode.className = "select xl-col-2 lg-col-2 md-col-3 sm-col-4 col-10";
-searchText.className =
-  "searchInText xl-col-2 lg-col-2 md-col-3 sm-col-3 col-10";
-searchState.className =
-  "searchState xl-col-4 lg-col-5 md-col-6 sm-col-7 col-12";
-
-rootElem.appendChild(searchDiv);
-rootElem.appendChild(episodeDiv);
-
-searchDiv.appendChild(selectShow);
-searchDiv.appendChild(selectEpisode);
-searchDiv.appendChild(searchText);
-searchDiv.appendChild(searchState);
-selectEpisode.style.display = "none";
-searchState.style.display = "none";
-
+//You can edit ALL of the code here
 function setup() {
-  let allShows = getAllShows();
-  console.log(allShows);
-  sortOn(allShows, "name");
-  makePageForShows(allShows);
-  addShow(allShows);
-  searchShows(allShows);
-}
-
-function sortOn(shows, name) {
-  //Alphabeticaly and case insensitive sort the shows by name.
-  shows.sort(function (a, b) {
-    return a[name].localeCompare(b[name]);
-  });
-  return shows;
-}
-
-function makePageForShows(showList) {
-  episodeDiv.innerHTML = "";
-  showList.forEach((show) => {
-    //Iterate through the array of episode.
-    let summaryText = show.summary.replace(/<p>/g, " "); //remove the '<p>' from the summary.
-    let pEl = summaryText.replace(/<\/p>/g, " "); //remove the '</p>' from the summary.
-
-    //Create the elements by innerHTML and add to episodeDiv div.
-    episodeDiv.innerHTML += `
```

Comment:
It is good and you got a lot of classes for the proper responsibility, but innerHTML is dangerous as you can fetch some scripts and insert them on your page. Try to play with  display:flex,  flex-direction: row and flex-wrap: wrap.  And width of elements set as min-width in px and width in VW. So it will be responsible.

Reply:
Yes it's right. I should use of create Element instead of innerHTML.